docs/src/pages/guides/styled-engine/styled-engine.md

```diff
@@ -1,21 +1,20 @@
-# Styled engine
+# Style library
```

Comment:
Should we also change the link + sidebar title?

Reply:
I think that we should keep the terminology consistent, for the h1, title, package name, URL folder, etc.

IMHO the existing title is less confusing than this new proposed one.

Reply:
@mbrookes the name "Styled engine" comes from the "the engine for the styled() util" :) What was the reasoning you had for changing it? Did it look like a typo? :)

Reply:
Yes, it appeared to be a reference to the generic term for style libraries - "style library". If it's a reference to the package, then I think it should use the package name `styled-engine` (as confused and confusing as that already is), not try to supplant recognised terminology with our own.

Reply:
+1 for `styled-engine`

Reply:
```suggestion
# Styled engine
```

Reply:
```suggestion
# `styled-engine`
```